lib/decorators/placeholders.test.js

```diff
@@ -14,6 +14,29 @@ describe('placeholders', () => {
     sandbox.restore();
   });
 
+  describe('isFieldEmpty', () => {
+    const fn = lib.isFieldEmpty;
+
+    it('throws error if path depth is greater than 2', () => {
+      expect(() => fn({}, 'one.two.three')).to.throw(Error);
```

Comment:
we should support n-depth nesting, since complex-lists may exists inside of other complex-lists